packages/patternfly-4/react-core/src/components/AlertGroup/AlertGroup.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { canUseDOM } from 'exenv';
+import { Omit } from '../../helpers/typeUtils';
+import { AlertGroupInline } from './AlertGroupInline';
+
+export interface AlertGroupProps extends Omit<React.HTMLProps<HTMLUListElement>, 'className'> {
+  /** Additional classes added to the AlertGroup */
+  className?: string;
+  /** Alerts to be rendered in the AlertGroup */
+  children?: React.ReactNode;
+  /** Toast notifications are positioned at the top right corner of the viewport */
+  isToast?: boolean;
+  /** Determine where the alert is appended to */
+  appendTo?: HTMLElement | (() => HTMLElement);
+}
+
+interface AlertGroupState {
+  container: HTMLElement;
+}
+
+export class AlertGroup extends React.Component<AlertGroupProps, AlertGroupState> {
+  state = {
+    container: undefined
+  } as AlertGroupState
+
+  componentDidMount() {
+    const container = document.createElement('div');
+    const target: HTMLElement = this.getTargetElement();
+    this.setState({ container });
+    target.appendChild(container);
+  }
+
+  componentWillUnmount() {
+    const target: HTMLElement = this.getTargetElement();
+    if (this.state.container) {
+      target.removeChild(this.state.container);
+    }
+  }
+
+  getTargetElement() {
+    const appendTo = this.props.appendTo;
+    if (typeof appendTo === 'function') {
+      return appendTo();
+    }
+    return appendTo || document.body;
+  };
+
+  render() {
+    const alertGroup = <AlertGroupInline {...this.props} />;
```

Comment:
Great changes! One suggestion:
Could you spread out the props that you need and pass them explicitly?
e.g. something like
```
const { className, children, isToast } = this.props;
const alertGroup = <AlertGroupInline className={className} isToast={isToast}>{children}</AlertGroupInline>
```
Then you also don't need the appendTo prop in AlertGroupInline